httpbakery/client_test.go

```diff
@@ -238,7 +238,7 @@ func assertDischargeServerDischargesConditionForVersion(c *gc.C, cond string, ve
 	}, nil)
 	c.Assert(err, gc.IsNil)
```

Comment:
I wonder if we should test whether the context actually gets through to the parts we intend it to.